### ISSUE TITLE
os_info.get().os_type() returns OracleLinux for a Centos machine

### ISSUE THREAD

**Issue by DurhamG** [DarkEld3r/os_info#161]
Using os_info version 2.0.2, I see a Centos 7 machine getting reported as OracleLinux.  It looks like 4f341a549dbcc32fdb5424af9c648f66baaafd95 added support for OracleLinux by checking if /etc/os-release exists, but it looks like that file exists for more than just OracleLinux.  file_release::retrieve() seems to loop through the distributions, and as soon as it finds a release who's file exists it uses it.  In this case, the retrieve loop should probably check the contents of the file before deciding to use it to confirm a particular distribution.

For reference, on Centos 7 my /etc/os-release looks like so:
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

**Comment by DarkEld3r** [comment]
Thanks for the report! I'm going to look into this issue, but if you would like to try to fix it, feel free to try.

**Comment by DarkEld3r** [comment]
Could you please check the fix from the master branch? I'm going to release a new version soon.

**Comment by DarkEld3r** [comment]
I have just published the `2.0.3` release with this fix.
